qa/os/src/test/java/org/elasticsearch/packaging/util/Docker.java

```diff
@@ -76,7 +77,11 @@
      * @param distribution details about the docker image to potentially load.
      */
     public static void ensureImageIsLoaded(Distribution distribution) {
-        final long count = sh.run("docker image ls --format '{{.Repository}}' " + distribution.flavor.name).stdout.split("\n").length;
+        Shell.Result result = sh.run("docker image ls --format '{{.Repository}}' " + distribution.flavor.name);
+
+        final long count = Arrays.stream(result.stdout.split("\n"))
```

Comment:
Figured I'd add the token "this could be simpler" comment 😉 

```
return Arrays.stream(result.stdout.split("\n"))
                .map(String::trim)
                .noneMatch(String::isEmpty);
```

Reply:
I'll break up the complicated filter, but I think I'll keep the `count` approach.